docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
         target: /var/lib/postgresql/data
     env_file:
       - ./.env.db
+    ports:
```

Comment:
shall not be part of productive docker-compose file. Use docker-compose.dev.xml instead to propagate (security reason)